src/windshaft/named-map.js

```diff
@@ -13,13 +14,18 @@ var NamedMap = MapBase.extend({
     //       "2": "/** torque visualization */\n\nMap { ... }"
     //     }
     //   }
-    //
-    json.styles = this._layersCollection.reduce(function (p, c, i) {
-      var style = c.get('cartocss');
+
+    var layers = this._layersCollection.filter(function (layer) {
+      return layer.get('type') !== 'GMapsBase';
```

Comment:
@nobuti I added [these little helpers](https://github.com/CartoDB/cartodb.js/blob/v4/src/geo/map/layer-types.js#L25-L27) to avoid typing layer types directly... I'd be 🔝  if we could use it here.